mmdeploy/codebase/mmdet/models/backbones.py

```diff
@@ -46,7 +46,7 @@ def focus__forward__ncnn(self, x):
 
     x = x.reshape(batch_size, c * h, 1, w)
     _b, _c, _h, _w = x.shape
-    g = _c // 2
+    g = torch.div(_c, 2, rounding_mode='floor')
```

Comment:
What's the benefit?

Reply:
deprecate warning

Reply:
Why not just rewrite `floordiv`?

Reply:
Rewriter should not change the behavior of the function.